crates/flux-refineck/src/fixpoint_encoding.rs

```diff
@@ -565,6 +595,59 @@ where
     fn def_span(&self) -> Span {
         self.genv.tcx.def_span(self.def_id)
     }
+
+    fn alias_pred_sort(arity: usize) -> rty::PolyFuncSort {
+        let mut sorts = vec![];
+        for i in 0..arity {
+            sorts.push(rty::Sort::Var(rty::SortVar::from(i)));
+        }
+        sorts.push(rty::Sort::Bool);
+        rty::PolyFuncSort::new(arity, rty::FuncSort { inputs_and_output: List::from_vec(sorts) })
```

Comment:
hmmm, this is making the function unnecessarily polymorphic. I don't think this will cause problems because well-formedness in previous phases will ensure the function is always instantiated with the same sorts, but technically, the more right thing to do would be to get the sort from the alias pred and encode that. Maybe we can leave a comment.

Reply:
Yes, I thought about that -- but then realized its easier (if slower) to kick the can down the road and let `fixpoint` work out the sorts, by just using a polymorphic signature for the UIF. But let me add your note as a comment for now.